Cesium3DTilesSelection/include/Cesium3DTilesSelection/TilesetOptions.h

```diff
@@ -157,7 +157,7 @@ struct CESIUM3DTILESSELECTION_API TilesetOptions {
    * unloaded until the total is under this number or until only required tiles
    * remain, whichever comes first.
    */
-  int64_t maximumCachedBytes = 512 * 1024 * 1024;
+  int64_t MaximumCachedBytes = 256 * 1024 * 1024;
```

Comment:
This should remain lowercase, consistent with the cesium-native coding standards.

Reply:
Done !

Reply:
should we still keep the default value 512 mb in cache since this property is now exposed in the Unreal side, so that user can adjust their cache depend on their use case? Changing it lower may worsen the performance for some users